katsdptelstate/telescope_state.py

```diff
@@ -110,11 +110,19 @@ def __init__(self, endpoint='', db=0, prefixes=(b'',), base=None):
         else:
             from .redis import RedisBackend
             if isinstance(endpoint, str) and '://' in endpoint:
-                r = redis.StrictRedis.from_url(endpoint, db=db, socket_timeout=5)
+                r = redis.StrictRedis.from_url(
```

Comment:
Should we switch to Redis rather than StrictRedis since that has been deprecated ?

Reply:
There are a few things I can clean once we drop redis-py 2.x, but I didn't want to conflate those with this emergency PR.

Reply:
Fair enough